hasura.planx.uk/migrations/1699449006468_setup_camden_domain/up.sql

```diff
@@ -0,0 +1,3 @@
+UPDATE teams
+SET domain = 'planningservices.camden.gov.uk'
+WHERE slug = 'camden';
```

Comment:
Are there any data sync side effects of handling this via migration? Eg it means it will get set locally & on staging, whereas I think `domain` column is omitted from data sync script?

Reply:
Very good point thanks! Migration dropped from PR in favour of doing this manually on prod only.